rasa/data.py

```diff
@@ -156,15 +155,7 @@ def _is_valid_filetype(path: Text) -> bool:
 
 
 def _is_nlu_file(file_path: Text) -> bool:
-    with open(file_path, encoding="utf-8") as f:
-        if file_path.endswith(".json"):
-            content = io_utils.read_json_file(file_path)
-            is_nlu_file = (
-                isinstance(content, dict) and content.get("rasa_nlu_data") is not None
-            )
-        else:
-            is_nlu_file = any(_contains_nlu_pattern(l) for l in f)
-    return is_nlu_file
+    return guess_format(file_path) != 'unk'
```

Comment:
use the defined constant here

Reply:
I added a constant called `NLU_FORMAT_UNKNOWN` with value "unk" to `constants.py` file because this same value is used in different files